__test__/integration/debt_token_api/scenarios/successful_transfer_from.ts

```diff
@@ -0,0 +1,117 @@
+// External
+import { BigNumber } from "bignumber.js";
+
+// Utils
+import * as Units from "utils/units";
+import { ERC20TokenSymbol } from "utils/constants";
+import { ACCOUNTS } from "../../../accounts";
+
+// Types
+import { DebtTokenScenario } from "./";
+import { SimpleInterestLoanOrder } from "src/adapters/simple_interest_loan_adapter";
+
+const CREDITOR = ACCOUNTS[0].address;
+const DEBTOR = ACCOUNTS[1].address;
+const TOKENS_APPROVED_OPERATOR = ACCOUNTS[2].address;
+const OWNERS_APPROVED_OPERATOR = ACCOUNTS[3].address;
+
+const ORDER_1: SimpleInterestLoanOrder = {
```

Comment:
can you use the data that we already have defined in the `ORDERS` namespace?

Reply:
Yes, fixing